documentation/release-process.md

```diff
@@ -96,4 +98,10 @@ The remainder of the release will automatically push NuGet packages to nuget.org
 1. The `dotnet-docker` team will merge from `nightly` branch to `main` branch and wait for `dotnet-monitor` team approval. Typically, these changes are completed the day before the release date.
```

Comment:
Can't leave a comment on the exact line, but for Step 1 I initially planned to include the appropriate email address - I decided I'd ask before adding it to avoid revealing an email address that should not be publicly available.

Reply:
I would not include that information here.